frontend/src/resources/utils/resource-request.ts

```diff
@@ -112,7 +112,7 @@ export async function reconcileResources(
             if (existing) {
                 const patch = jsonpatch.compare(existing, resource)
                 if (patch.length) {
-                    await patchResource(existing, patch, { dryRun: true })
+                    patchResource(existing, patch, { dryRun: true })
```

Comment:
Looks like you removed an `await` that you had not added. I think this await is intentional so that we don't proceed with deletion of any resources unless patching succeeds. Other instances of this in this file to revert as well.